.github/workflows/docker-publish.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     - name: Publish to Registry
       uses: elgohr/Publish-Docker-Github-Action@master
       with:
-        name: pcic/climate-explorer-frontend
+        name: pcic/plan2adapt-v2-frontend
```

Comment:
Crap, to think I didn't spot that!

Reply:
Luckily I noticed it while helping Nic out with an action of his own!